dependencies.gradle

```diff
@@ -0,0 +1,20 @@
+ext {
+    androidCompileSdkVersion = 23
```

Comment:
We have these inside `versions` array in our project. 
Much similar to this. https://github.com/mr-archano/Playground/blob/develop/gradle/dependencies.gradle

I really like it.